src/index.js

```diff
@@ -1,41 +1,126 @@
-// eslint-disable-next-line no-unused-vars
-const tasks = [
-  {
-    description: 'center a div',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'build a submit button',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'build a form',
-    completed: false,
-    index: 3,
-  },
-];
-
-// eslint-disable-next-line no-unused-vars
+// eslint-disable-next-line import/extensions
+import { AddStore } from './modules/addstore.js';
+// eslint-disable-next-line import/extensions
+import {
+  addToLocalStorage,
+  removeFromLocalStorage,
+  // eslint-disable-next-line import/extensions
+} from './modules/storesave.js';
+import './styles/correct.css';
+
 const todoInput = document.querySelector('.todo-input');
-const todoButton = document.querySelector('.todo-button');
-const todoList = document.querySelector('#todo-list');
-const clearButtonDiv = document.querySelector('.clear-button');
+const addButton = document.querySelector('.add-button');
+const todoMenu = document.querySelector('.todo-menu');
+const filterTasks = document.querySelector('.filter-tasks');
+const clearButton = document.querySelector('.clear-button');
+const newTask = document.createElement('li');
+
+document.addEventListener('DOMContentLoaded', AddStore);
+
+filterTasks.addEventListener('click', (e) => {
+  const tasks = document.querySelectorAll('.todo');
+  tasks.forEach((task) => {
+    // eslint-disable-next-line default-case
+    switch (e.target.value) {
+      case 'all':
+        task.style.display = 'block';
+        break;
+      case 'completed':
+        if (task.classList.contains('completed')) {
+          task.style.display = 'block';
+        } else {
+          task.style.display = 'none';
+        }
+    }
+  });
+});
 
-const addTodo = (e) => {
+clearButton.addEventListener('click', () => {
+  const tasks = document.querySelectorAll('.todo');
+  tasks.forEach((task) => {
+    if (task.classList.contains('completed')) {
+      task.style.display = 'none';
+    }
+  });
+});
+
+// Function to Add Tasks
+addButton.addEventListener('click', (e) => {
   e.preventDefault();
 
-  // deleteCompleted Button
-  const trashButton = document.createElement('button');
+  // Todo Div
+  const todoDiv = document.createElement('div');
+  todoDiv.classList.add('todo');
+
+  // Create New li
+  // const newTask = document.createElement('li')
+  newTask.innerText = todoInput.value;
+
+  todoInput.setAttribute('readonly', 'readonly');
+
+  newTask.classList.add('todo-item');
+
+  // Add the new li to todoDiv
+  todoDiv.appendChild(newTask);
+  addToLocalStorage(todoInput.value);
```

Comment:
- [ ] The new todos added to the local storage are returning null after the first item has been submitted. Ensure that the data you are storing in the local storage, shows the **description, completed and index** attributes. Please check it out 

![image](https://user-images.githubusercontent.com/63473584/179373480-d42da516-8177-4545-be08-433d597c27eb.png)